app/src/renderer/components/monitor/PageBlockchain.vue

```diff
@@ -1,11 +1,15 @@
 <template lang="pug">
```

Comment:
I still can't see the page. It references state.blockchain that is not set anywhere (or I can't find it).

Reply:
This is resolved in a different PR #113

Reply:
The network monitor is on @mappum's plate. But #113 will show that the data is not loading yet.